cmd/viewer/viewer.go

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Viewer is a tool to automate the creation of read-only wrapper structs.
```

Comment:
of "view" wrapper types that provide read-only accessor methods to the underlying type

or something